provider/resources.go

```diff
@@ -213,7 +213,7 @@ func docEditRules(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
 
 // The Authentication section contains links to internal guides.
 var skipAuthenticationSection = tfbridge.DocsEdit{
-	Path: "index.md",
+	Path: "index.html.markdown",
```

Comment:
I'm curious about this change, since it didn't seem to affect what's rendered to _index.md

Reply:
This happened because on these early rollouts, the generator rules were accidentally being called on _all_ files, so the filename turned out to not matter at all.
This has been fixed since.